Stewart et al. Supplementary Code.Rmd

```diff
@@ -1,243 +1,252 @@
----
-title: "Replication document for Extreme Events Review"
-author: "Mathew Stewart"
-output: pdf_document
----
-
-## Load Libraries
-```{r, libraries, message=FALSE}
-library(tidyverse)
-library(vegan)
-library(tidytext)
-library(gplots)
-library(reshape2)
-```
-
-## Set working directory and load data
-```{r, setup}
-setwd("/Users/stewart/Dropbox/Projects/Extreme events/Quant review/Data/")

```

Comment:
I removed this right away because using `setwd()` immediately makes code unreproducible. And if you use an RStudio R Project combined with the `here()` function when referencing files, it fixes this issue.